tests/bluetooth/controller/common/src/helper_util.c

```diff
@@ -0,0 +1,259 @@
+/*
+ * Copyright (c) 2020 Demant
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "zephyr/types.h"
+#include "ztest.h"
+#include <stdlib.h>
+#include "kconfig.h"
+
+#include <bluetooth/hci.h>
+#include <sys/slist.h>
+#include <sys/util.h>
+
+#include "hal/ccm.h"
+
+#include "util/mem.h"
+#include "util/memq.h"
+
+#include "pdu.h"
+#include "ll.h"
+#include "ll_settings.h"
+#include "ll_feat.h"
+
+#include "lll.h"
+#include "lll_conn.h"
+
+#include "ull_conn_types.h"
+#include "ull_tx_queue.h"
+#include "ull_llcp.h"
+
+#include "helper_pdu.h"
+#include "helper_util.h"
+
+static u16_t lazy;
+sys_slist_t ut_rx_q;
+static sys_slist_t lt_tx_q;
+
+
+#define PDU_DC_LL_HEADER_SIZE	(offsetof(struct pdu_data, lldata))
+#define NODE_RX_HEADER_SIZE	(offsetof(struct node_rx_pdu, pdu))
+#define NODE_RX_STRUCT_OVERHEAD	(NODE_RX_HEADER_SIZE)
+#define PDU_DATA_SIZE		(PDU_DC_LL_HEADER_SIZE + LL_LENGTH_OCTETS_RX_MAX)
+#define PDU_RX_NODE_SIZE WB_UP(NODE_RX_STRUCT_OVERHEAD + PDU_DATA_SIZE)
+
+helper_pdu_encode_func_t * const helper_pdu_encode[] = {
+	helper_pdu_encode_version_ind,
+	NULL,
+	helper_pdu_encode_reject_ext_ind,
+	helper_pdu_encode_enc_req,
+	helper_pdu_encode_enc_rsp,
+	helper_pdu_encode_start_enc_req,
+	helper_pdu_encode_start_enc_rsp,
+	helper_pdu_encode_phy_req,
+	helper_pdu_encode_phy_rsp,
+	helper_pdu_encode_phy_update_ind,
+	helper_pdu_encode_unknown_rsp,
+};
+
+helper_pdu_verify_func_t *const helper_pdu_verify[] = {
+	helper_pdu_verify_version_ind,	  helper_pdu_verify_reject_ind,
+	helper_pdu_verify_reject_ext_ind, helper_pdu_verify_enc_req,
+	helper_pdu_verify_enc_rsp,	  helper_pdu_verify_start_enc_req,
+	helper_pdu_verify_start_enc_rsp,  helper_pdu_verify_phy_req,
+	helper_pdu_verify_phy_rsp,	  helper_pdu_verify_phy_update_ind,
+	helper_pdu_verify_unknown_rsp,
+};
```

Comment:
Can we do one entry per line?

Reply:
Yes, I'll do, not sure how it got formatted this way since the original was one entry per line.